### ISSUE TITLE
_gmaps_cdv_ Class Present After Calling Remove Method

### ISSUE THREAD

**Issue by mino922** [mapsplugin/cordova-plugin-googlemaps#2011]
**I'm submitting a ...**  (check one with "x")
- [ ] question
- [X] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [ ] **Android**
- [X] **iOS**

**cordova information:**  (run `$> cordova plugin list`) 

```
com.googlemaps.ios 2.5.0 "Google Maps SDK for iOS"
cordova-background-geolocation-lt 2.9.1 "BackgroundGeolocation"
cordova-plugin-actionsheet 2.3.3 "ActionSheet"
cordova-plugin-advanced-http 1.9.1 "Advanced HTTP plugin"
cordova-plugin-app-event 1.2.1 "Application Events"
cordova-plugin-background-fetch 5.1.1 "CDVBackgroundFetch"
cordova-plugin-background-mode 0.7.2 "BackgroundMode"
cordova-plugin-camera 2.4.1 "Camera"
cordova-plugin-cocoalumberjack 0.0.4 "CocoaLumberjack"
cordova-plugin-compat 1.2.0 "Compat"
cordova-plugin-console 1.1.0 "Console"
cordova-plugin-device 1.1.7 "Device"
cordova-plugin-dialogs 2.0.1 "Notification"
cordova-plugin-facebook4 1.9.1 "Facebook Connect"
cordova-plugin-file 5.0.0 "File"
cordova-plugin-file-transfer 1.7.0 "File Transfer"
cordova-plugin-geolocation 3.0.0 "Geolocation"
cordova-plugin-googlemaps 2.2.0-beta-20180120-0115 "cordova-plugin-googlemaps"
cordova-plugin-inappbrowser 1.7.2 "InAppBrowser"
cordova-plugin-nativestorage 2.2.2 "NativeStorage"
cordova-plugin-network-information 1.3.4 "Network Information"
cordova-plugin-splashscreen 4.1.0 "Splashscreen"
cordova-plugin-statusbar 2.4.1 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-sqlite-storage 2.2.0 "Cordova sqlite storage plugin"
de.appplant.cordova.plugin.local-notification 0.8.5 "LocalNotification"
ionic-plugin-keyboard 2.2.1 "Keyboard"
uk.co.workingedge.phonegap.plugin.launchnavigator 4.1.3 "Launch Navigator"
```

**If you use `@ionic-native/google-maps`, please tell the package.json (only `@ionic-native/core` and `@ionic-native/google-maps` are fine mostly)**

**Current behavior:**
After I call the remove method of the map class, I still see the _gmaps_cdv_ class present and it is causing the background of my app to turn white. If I inspect and disable the class my app looks as expected.

**Expected behavior:**
After destroying the map all elements should be completely removed.

**Comment by wf9a5m75** [comment]
Thank you for reporting. Fixed the bug. Please reinstall the plugin from the multiple_maps branch.

**Comment by mino922** [comment]
Thank you. I confirmed the fix.

**Comment by gchokeen** [comment]
Is this issue merged in master branch yet? I'm using 2.4.6 but facing same issue?

**Comment by gchokeen** [comment]
This issue still seems not fixed, I tried using multiple_maps branch

`    <plugin name="cordova-plugin-googlemaps" spec="git+https://github.com/mapsplugin/cordova-plugin-googlemaps.git#multiple_maps">
`
Hack solution 

> 
> ionViewDidLeave(){
>      this.map.remove();
>      let body = document.getElementsByTagName('body')[0];
>      body.classList.remove("_gmaps_cdv_");
>      let appRoot = document.getElementsByTagName('ion-app')[0];
>      appRoot.classList.remove("_gmaps_cdv_");
>    }
